test/e2e/data/infrastructure-aws/withoutclusterclass/kustomize_sources/default/kustomization.yaml

```diff
@@ -1,6 +1,25 @@
+# default template enables external ccm and csi
```

Comment:
Hi @Ankitasw, 
Default template enables external ccm and csi. Templates for other flavors are based on this default template and then patch some more.
So we don't need the external-cloud-provider template but we need an intree-cloud-provider template for the CCM/CSI upgrade tests.

Reply:
Got it 👍